R/guide-bins.R

```diff
@@ -178,7 +178,7 @@ guide_train.bins <- function(guide, scale, aesthetic = NULL) {
     limits <- all_breaks[c(1, length(all_breaks))]
     breaks <- all_breaks[-c(1, length(all_breaks))]
   }
-  key <- new_data_frame(setNames(list(c(scale$map(bin_at), NA)), aes_column_name))
+  key <- data_frame(c(scale$map(bin_at), NA), .name_repair = ~ aes_column_name)
```

Comment:
What you had before with `vctrs::new_data_frame()` on a named list seems clearer to me than using name repair this way

Reply:
interesting - I find the other way much more readable as it avoids multiple nested calls